test/katex-spec.js

```diff
@@ -2829,7 +2884,7 @@ describe("Unicode accents", function() {
             "\\tilde n" +
             "\\grave o\\acute o\\hat o\\tilde o\\ddot o" +
             "\\grave u\\acute u\\hat u\\ddot u" +
-            "\\acute y\\ddot y");
+            "\\acute y\\ddot y", nonstrictSettings);
```

Comment:
I noticed you added `nonstrictSettings` in a couple of places.  Does this modify the behaviour fo the test?  How is it related to the `\newcommand`, `\renewcommand`, etc. part of this PR?

Reply:
Before, we used `{strict: false}` in all tests.  Now we use the default settings `{}` in all tests by default, which is more natural.  A few tests needed to have `{strict: false}` specified, though, and I added `nonstrictSettings` to these.